avatars/models.py

```diff
@@ -170,19 +180,34 @@ class AvatarizationResultResponse(BaseModel):
     privacy_metrics: PrivacyMetrics
     signal_metrics: SignalMetrics
     avatars_dataset: DatasetResponse
-    sensitive_unshuffled_avatars_datasets: DatasetResponse
+    sensitive_unshuffled_avatars_dataset: DatasetResponse
 
 
 class Job(BaseModel):
     id: UUID = Field(..., title="Id")
     status: JobStatus
-    dataset_id: UUID = Field(..., title="Dataset Id")
+    dataset_id: Optional[UUID] = Field(
+        None,
+        description="Deprecated, use parameters.dataset_id instead.",
+        title="Dataset Id",
+    )
     error_message: Optional[str] = Field(None, title="Error Message")
     traceback: Optional[str] = Field(None, title="Traceback")
     result: Optional[AvatarizationResultResponse] = None
+    parameters: AvatarizationParameters
     current_progress: Optional[JobProgress] = None
 
 
 class JobCreate(BaseModel):
```

Comment:
Why don't you call `AvatarizationJobCreate` in this class (to make it backward compatible)?

Reply:
The class is just a model to validate data, it doesn't make any calls. I just chose to keep the previous stuff but delete it from the tutorial, marking it as deprecated and putting the correct method in the tutorial.